latest version has a bug

### Describe the bug

This error happened while installing the dependencies of wxt@0.20.11
 at @wxt-dev/storage@1.2.5

### Reproduction

just run installation

```
pnpm dlx wxt@latest 
```

### Steps to reproduce

just run installation

```
pnpm dlx wxt@latest 
```

### System Info

```shell
System:
    OS: macOS 26.0.1
    CPU: (8) arm64 Apple M3
    Memory: 142.61 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 24.8.0 - /opt/homebrew/bin/node
    npm: 11.6.0 - /opt/homebrew/bin/npm
    pnpm: 10.16.1 - /opt/homebrew/bin/pnpm
 
  Browsers:
    Chrome: 140.0.7339.214
    Firefox: 143.0.4
    Safari: 26.0.1
```

### Used Package Manager

pnpm

### Validations

- [x] Read the [Contributing Guidelines](https://github.com/wxt-dev/wxt/blob/main/CONTRIBUTING.md).
- [x] Read the [docs](https://wxt.dev/guide/installation.html).
- [x] Check that there isn't [already an issue](https://github.com/wxt-dev/wxt/issues) that reports the same bug to avoid creating a duplicate.
- [x] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/wxt-dev/wxt/discussions) or join our [Discord Chat Server](https://discord.gg/ZFsZqGery9).
- [x] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

This issue is happening in all package managers

### PNPM
```powershell
PS C:\Users\Loc\Projects> pnpm dlx wxt@latest init
C:\Users\Loc\AppData\Local\pnpm-cache\dlx\df4aa4bb86bdf0ec6ad2465aca479d619d353227517c083cc6d576fb80fc73e9\199edb6d7ef-335c:
 ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In ..\AppData\Local\pnpm-cache\dlx\df4aa4bb86bdf0ec6ad2465aca479d619d353227517c083cc6d576fb80fc73e9\199edb6d7ef-335c: "@wxt-dev/browser@workspace:^" is in the dependencies but no package named "@wxt-dev/browser" is present in the workspace

This error happened while installing the dependencies of wxt@0.20.11
 at @wxt-dev/storage@1.2.5

Packages found in the workspace:
Progress: resolved 45, reused 45, downloaded 0, added 0
PS C:\Users\Loc\Projects>
```

### Bun
```powershell
PS C:\Users\Loc\Projects> bunx wxt@latest init
error: Script not found "wxt@latest"
PS C:\Users\Loc\Projects> 
```

### NPM
```powershell
PS C:\Users\Loc\Projects> npx wxt@latest init

WXT 0.20.11                                                                                                   16:57:56
ℹ Initializing new project                                                                                   16:57:56
√ Project Directory ... test
√ Choose a template » vue
√ Package Manager » npm
✔ Downloading template

✨ WXT project created with the vue template.                                                                 16:58:02

Next steps:                                                                                                   16:58:02
  1. cd test                                                                                                  16:58:02
  2. npm install                                                                                              16:58:02

PS C:\Users\Loc\Projects> cd .\test\
PS C:\Users\Loc\Projects\test> npm install
npm error code EUNSUPPORTEDPROTOCOL
npm error Unsupported URL Type "workspace:": workspace:^
npm error A complete log of this run can be found in: C:\Users\Loc\AppData\Local\npm-cache\_logs\2025-10-16T15_58_06_665Z-debug-0.log
PS C:\Users\Loc\Projects\test>
```

### NPX (using PNPM as package manager)
```powershell
PS C:\Users\Loc\Projects> npx wxt@latest init

WXT 0.20.11                                                                                                   16:54:54
ℹ Initializing new project                                                                                   16:54:54
√ Project Directory ... test
√ Choose a template » vue
√ Package Manager » pnpm
✔ Downloading template

✨ WXT project created with the vue template.                                                                 16:55:12

Next steps:                                                                                                   16:55:12
  1. cd test                                                                                                  16:55:12
  2. pnpm install                                                                                             16:55:12

PS C:\Users\Loc\Projects> cd test
PS C:\Users\Loc\Projects\test> pnpm install
 ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In : "@wxt-dev/browser@workspace:^" is in the dependencies but no package named "@wxt-dev/browser" is present in the workspace

This error happened while installing the dependencies of wxt@0.20.11
 at @wxt-dev/storage@1.2.5

Packages found in the workspace:
Progress: resolved 81, reused 81, downloaded 0, added 0
PS C:\Users\Loc\Projects\test>
```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -62,8 +62,7 @@ jobs:
       - name: Publish to NPM
         working-directory: packages/${{ inputs.package }}
         run: |
-          sudo npm i -g npm@latest
-          /usr/local/bin/npm publish
+          pnpm publish
 
       - name: Create GitHub release
         run: pnpm tsx scripts/create-github-release.ts ${{ inputs.package }}

